var/spack/repos/builtin/packages/librttopo/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Librttopo(Package):
```

Comment:
I am not able to build this package as written.  

Why not inherit from `AutotoolsPackage`?

```suggestion
class Librttopo(AutotoolsPackage):
```

Or you need to implement an `install` method.